static/js/directives/auth-group.js

```diff
@@ -0,0 +1,53 @@
+var _ = require('underscore'),
+    asyncSome = require('async/some');
+
+// Hides an element based on an array of expressions, when none of the provided expressions evaluates to Boolean true.
+// If any of the expression values is a String or an Array, they will be interpreted as permissions and passed to Auth
+// service for evaluation.
+
+angular.module('inboxDirectives').directive('mmAuthGroup', function(Auth) {
```

Comment:
Maybe this should be named `mmAuthAny` to make it clear that the user just needs one of the permissions?

Reply:
I'm also wondering if there's some way to combine the `auth` and `authGroup` directives...

Reply:
Finally, I think this code should be in the Auth service (eg: `auth.any()`, `auth.all()`) so it can be used by the rest of the codebase, but also because it should be more performant to call the Auth service once rather than getting the users roles _n_ times.